hat/app/org/hatdex/hat/she/mappers/DataEndpointMapper.scala

```diff
@@ -48,6 +48,8 @@ trait DataEndpointMapper extends JodaWrites with JodaReads {
       fromDate: Option[DateTime],
       untilDate: Option[DateTime]): Option[FilterOperator.Operator] =
     fromDate.map { date =>
+      logger.info(s"fromDate ${fromDate}")
```

Comment:
remove these four loggers when you can please.

Reply:
Done.